deploy/crds/launcher_v1alpha1_launcher_cr.yaml

```diff
@@ -2,8 +2,43 @@ apiVersion: launcher.fabric8.io/v1alpha1
 kind: Launcher
 metadata:
   name: example-launcher
+  environment: production
 spec:
-  github:
-    username: username
-    token: token
+  catalog:
+    repositoryUrl: http://github.com/fabric8-launcher/launcher-booster-catalog
+    repositoryRef: master
+    filter: filter expression
+    reindexToken: token
   creator: false
+  openshift:
+    apiUrl: apiUrl (default is the same cluster as the operator is installed)
+    consoleUrl: consoleUrl
+    subscriptionToken: token
+    auth:
+      username: username
+      password: password
+      token: token
+      impersonate: true
+    clusters:
+  git:
+    provider: GITHUB
+    url: https://github.com
+    repositoryDescription: Generated by the Red Hat Developer Launch (https://developers.redhat.com/launch)
+    auth:
```

Comment:
we could directly put `username` and `token` in `git` and remove the `auth` layer.